cefbrowser/cefbrowser.py

```diff
@@ -304,25 +348,51 @@ def delete_cookie(self, url=""):
             Logger.warning("No cookie manager found!, Can't delete cookie(s)")
 
     def on_url(self, instance, value):
-        if self._browser and value and value!=self._browser.GetUrl():
-            #print("ON URL", instance, value, self._browser.GetUrl(), self._browser.GetMainFrame().GetUrl())
+        if self._browser and value and value != self._browser.GetUrl():
+            # print(
+            #     "ON URL",
+            #     instance,
+            #     value,
+            #     self._browser.GetUrl(),
+            #     self._browser.GetMainFrame().GetUrl(),
+            # )
             self._browser.Navigate(self.url)
 
-    def on_js_dialog(self, browser, origin_url, accept_lang, dialog_type, message_text, default_prompt_text, callback,
-                     suppress_message):
+    def on_js_dialog(
+        self,
+        browser,
+        origin_url,
+        accept_lang,
+        dialog_type,
+        message_text,
+        default_prompt_text,
+        callback,
+        suppress_message,
```

Comment:
In case you haven't seen, you can now remove unused args from callbacks signature and use **kwargs, see: https://github.com/cztomczak/cefpython/blob/master/docs/Migration-guide.md#v553-handlers-callbacks-and-other-interfaces